shared/chat/conversation/messages/attachment/image/index.js

```diff
@@ -31,8 +31,10 @@ type Props = {
   progress: number,
   progressLabel: string,
   showPlayButton: boolean,
+  showFilmButton: boolean,
```

Comment:
should likely be something
`showButton: null | 'play' | 'film'`
since its really a state and we'd never show both at the same time